v1.5/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 data:
   # The location of your etcd cluster.  This uses the Service clusterIP
   # defined below.
-  etcd_endpoints: "http://100.78.232.136:6666"
+  etcd_endpoints: "http://10.96.232.136:6666"
```

Comment:
also, kubeadm (latest) guide links to Calico v1.5 docs pointing to this doc which won't work for the latest kubeadm